internal/pkg/deploy/cloudformation/stack/rd_web_svc.go

```diff
@@ -141,7 +141,7 @@ func (s *RequestDrivenWebService) Template() (string, error) {
 			Tracing: strings.ToUpper(aws.StringValue(s.manifest.Observability.Tracing)),
 		},
 		PermissionsBoundary:  s.permBound,
-		Private:              !s.manifest.Private.IsZero(),
```

Comment:
Would this work if we address this comment:
https://github.com/aws/copilot-cli/pull/4164#discussion_r1022136746

Is it worthwhile to cut that PR first before we merge this PR or #4164 given that we now have 2 places that would benefit from this change?